src/app/progress/SeasonalRank.tsx

```diff
@@ -64,6 +64,16 @@ export default function SeasonalRank({
   const { progressToNextLevel, nextLevelAt } = prestigeMode ? prestigeProgress : seasonProgress;
   const { rewardItems } = defs.Progression.get(seasonPassProgressionHash);
 
+  const brightEngramRewardLevelsTemp: number[] = [];
+
+  rewardItems
+    .filter((item) => item.itemHash === brightEngramHash)
+    .forEach((item) => {
+      brightEngramRewardLevelsTemp.push(item.rewardedAtProgressionLevel % 10);
+    });
```

Comment:
```suggestion
  const brightEngramRewardLevelsTemp = rewardItems
    .filter((item) => item.itemHash === brightEngramHash)
    .map((item) => item.rewardedAtProgressionLevel % 10);
```
think this should have the same result.

Reply:
awesome